tests/accessor/accessor_default_values.h

```diff
@@ -0,0 +1,149 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  This file provides functions for tests on accessor default values.
+//
+*******************************************************************************/
+#ifndef SYCL_CTS_ACCESSOR_DEFAULT_VALUES_H
+#define SYCL_CTS_ACCESSOR_DEFAULT_VALUES_H
+
+#include "accessor_common.h"
+#include "catch2/catch_test_macros.hpp"
+
+#include <type_traits>
+
+namespace accessor_default_values_test {
+
+constexpr int expected_dims = 1;
+constexpr sycl::target expected_target = sycl::target::device;
+constexpr sycl::access::placeholder expected_placeholder =
+    sycl::access::placeholder::false_t;
```

Comment:
Just as a suggestion:
```suggestion
namespace detail {
constexpr int expected_dims = 1;
constexpr auto expected_target = sycl::target::device;
constexpr auto expected_placeholder = sycl::access::placeholder::false_t;
}  // namespace detail
```
By using `detail` namespace we are saying that we are not going to use these values outside of this header

Reply:
+1

Reply:
namespace `detail` has been added [here](https://github.com/KhronosGroup/SYCL-CTS/pull/335/commits/c040cc7560db878fb467d794a504c6e5e20a0d0e#diff-9ed3eae8f2889bc04dfc0fad247a8fa4e381e1fff4144e0f1a23670d108f6fe9R17)